evm-development/quickstart-guide.md

```diff
@@ -16,9 +16,27 @@ _Explorer:_ [_https://evm.explorer.canto.io/_](https://evm.explorer.canto.io/)
 
 <summary><strong>Alternative RPC URLs</strong></summary>
 
-* `https://canto.neobase.one/`
-* `https://canto.evm.chandrastation.com/`
-* `https://jsonrpc.canto.nodestake.top`
+* [https://canto.slingshot.finance](https://canto.slingshot.finance)
```

Comment:
Could you remove the Slingshot RPC from this list? It's listed right above the `summary` as the default RPC.